scss/_base_typography-definitions.scss

```diff
@@ -35,6 +35,20 @@
     font-weight: 550;
   }
 
+  %vf-heading-display {
+    @extend %vf-heading-2;
+
+    font-size: #{map-get($font-sizes, display-mobile)}rem;
+    font-weight: 100;
+    line-height: map-get($line-heights, display-mobile);
+    padding-top: 0;
```

Comment:
```suggestion
    padding-top: 0.4rem;
```

Reply:
and margin bottom to match - sp-after same as the others, minus the padding-top

Reply:
Mobile version seems to be perfectly on line without any padding.

Could you verify latest demo @lyubomir-popov ?
https://vanilla-framework-4840.demos.haus/docs/examples/patterns/headings/display